<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TcpProducer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="spacing">
       <number>3</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLineEdit" name="editIp">
          <property name="placeholderText">
           <string>IP do servidor</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="btnConnect">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Conectar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnDisconnect">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Desconectar</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="labelMin">
          <property name="text">
           <string>Valor mínimo:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QSlider" name="sliderMin">
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdMin"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="labelMax">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Valor máximo:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>6</number>
          </property>
          <item>
           <widget class="QSlider" name="sliderMax">
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdMax"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="labelTiming">
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="text">
           <string>Tempo (s):</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QSlider" name="sliderTiming">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdTiming">
            <property name="smallDecimalPoint">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="btnStart">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Começar</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnStop">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Parar</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QListView" name="listView"/>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
        <property name="textVisible">
         <bool>true</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
        <property name="format">
         <string>%p%</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>sliderMin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdMin</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderMax</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdMax</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>113</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderTiming</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdTiming</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editIp</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>setIpAddr(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderMin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>setMin(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>40</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderMax</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>setMax(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>34</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>44</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setIpAddr(QString)</slot>
  <slot>setMin(int)</slot>
  <slot>setMax(int)</slot>
 </slots>
</ui>
